dockers/docker-orchagent/mirror.json.j2

```diff
@@ -1,24 +0,0 @@
-[
```

Comment:
how are we going to create mirror session in the first place?

Reply:
It will be created by sonic-cfggen utility. In the same way as ACL table.